README.md

```diff
@@ -46,27 +46,6 @@ To deploy the Datadog Agent on hosts, add the Datadog role and your API key to y
 
 The API key is required and its absence causes the role to fail. If you want to provide it through another way, outside of Ansible's control, specify a placeholder key and substitute the key at a later point.
 
-### Air-gapped environments
-
-To install Datadog in an air-gapped environment using a specific registry and mirrored images, use the Datadog Ansible collection along with the `datadog_installer_registry`, `datadog_installer_auth`, and `agent_datadog_config` variables. While Datadog recommends fetching `latest`, `datadog_installer_version` and `datadog_apm_inject_version` can be provided to pin versions.
```

Comment:
If we're removing this documentation, does that mean the new OCI installer does not support air-gapped environments?

Reply:
it should, but the setup on customer side is slightly different, they also need to host the ssi script now. Moreover, considering it's advanced/complex, I elected to remove it from the readme